sample-app/src/components/InputDropdown.tsx

```diff
@@ -9,13 +9,18 @@ interface Props {
   screenReaderInstructionsId: string,
   screenReaderText: string,
   onSubmit?: (value: string) => void,
+  renderSearchButton?: () => JSX.Element | null,
+  renderLogo?: () => JSX.Element | null,
+  renderResultIcon?: () => JSX.Element | null,
```

Comment:
What would you think of moving this prop down to the section-level? Each section could define its own result icons.

Reply:
sure, that makes sense

Reply:
This prop actually was actually put in here by mistake. Currently the Options which are passed to the DropdownSection define the rendering of the result icon